WatchdogService/CleanupOperations.cs

```diff
@@ -270,140 +271,23 @@ internal async Task<int> EnumerateTableItemsAsync(CloudTable table)
 
             Stopwatch sw = Stopwatch.StartNew();
             ServiceEventSource.Current.Trace("EnumerateTableItemsAsync", table.Name);
+            TableClient tableClient = client.GetTableClient(table.Name);
 
-            TableBatchOperation tbo = new TableBatchOperation();
-
-            string pKey = null;
             int deleteCount = 0;
-            TableContinuationToken tct = null;
-            TableQuery query =
-                new TableQuery().Where(
-                    TableQuery.GenerateFilterConditionForDate("Timestamp", QueryComparisons.LessThan, DateTimeOffset.Now.Subtract(this._timeToKeep)));
+            var timeStamp = DateTimeOffset.Now.Subtract(this._timeToKeep);
+            // Execute the query.
+            AsyncPageable<TableEntity> queryResult = tableClient.QueryAsync<TableEntity>(filter: $"Timestamp lt '{timeStamp}'");
```

Comment:
We could use filter expression syntax here
```suggestion
            AsyncPageable<TableEntity> queryResult = tableClient.QueryAsync<TableEntity>(e => e.Timestamp < timeStamp);
```